src/promptflow-core/promptflow/core/_flow.py

```diff
@@ -417,6 +421,19 @@ def __call__(self, *args, **kwargs):
             outputs=self._outputs,
         )
 
+    def render(self, *args, **kwargs):
+        """Render the prompt content.
+
+        :param args: positional arguments are not supported.
+        :param kwargs: prompty inputs with key word arguments.
+        :return: Prompt content
+        :rtype: Union[str, list]
```

Comment:
when will this return a str?

Reply:
when the api type is completion, return vlaue is a string.